src/pudl_archiver/zenodo/api_client.py

```diff
@@ -292,136 +236,19 @@ async def _apply_changes(self):
             logger.info("Dry run, aborting.")
             return
         for file_info in self.deletes:
-            await self.delete_file(file_info)
+            await self.depositor.delete_file(self.new_deposition, file_info.filename)
```

Comment:
I spent a long time trying to work out why these were giving me 403's - turns out I was targeting the `self.deposition`, which had already been published 🪦